Unity-Technologies/Graphics
# Platform/gamecore

# **Please read the [Contributing guide](CONTRIBUTING.md) before making a PR.**

* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker

---
### Purpose of this PR
This PR adds support for the XboxSeries platform to the SRP packages and the PPv2 package.

---
### Testing status
Manual testing with HDRP packages
Testing of HDRP, URP, ShaderGraph and VFXGraph run locally.
Tested on Xbox to check for regressions

---
### Comments to reviewers
PPV2 changes

The Xbox build target is wrapped to allow this package to work even in older versions of Unity as PS5 will not show up as a valid enum member there.
